src/App.tsx

```diff
@@ -37,6 +38,8 @@ function App() {
         { path: 'verifyAccount', element: <VerifyAccount /> },
```

Comment:
The name of the paths should be complete and sepearted with hyphens ex: 
```
verify-account
forget-password
```

We shouldn't include the header in the pages as well.